elements/pfe-sass/variables/_maps.scss

```diff
@@ -7,6 +7,7 @@ $pfe-colors: (
   text:                               $pfe-color--text,
   text--on-dark:                      $pfe-color--text--on-dark,
   text--on-saturated:                 $pfe-color--text--on-saturated,
+  text--deemphasized:                 $pfe-color--gray-500,
```

Comment:
Any chance that ui-disabled--text meets this need?  They seem really similar to me?

Reply:
+1

Reply:
@castastrophe removed this variable name.